modules/internal/String.chpl

```diff
@@ -41,10 +41,16 @@ private module BaseStringType {
 // It might be worth moving them here for documentation - KB Feb 2016
 
 /*
-  The following documentation consists of functions and methods used to
+  The following documentation shows functions and methods used to
   manipulate and process Chapel strings. See the :mod:`IO` documentation for
   more information on reading, writing, and formatting strings.
```

Comment:
I like these changes. Should this sentence about the IO documentation be moved or removed?